### ISSUE TITLE
Creating a database named 0 gives a fatal error

### ISSUE THREAD

**Issue by nulll-pointer** [phpmyadmin/phpmyadmin#14314]
### Steps to reproduce
1. Click new database in the navigation.
2. Enter 0 in the database name field and submit.

### Expected behaviour
Should create database 0/give an sql error.

### Actual behaviour
Gives a fatal error

Note: I'm not sure about the expected behaviour, because, creating a database 00 gives error in mysql, but not in phpmyadmin, maybe because it uses backquotes while inserting.
### Server configuration
ubuntu 16.04

**Web server:**
Apache/2.4.29 

**Database:**
mariadb 10.1.30

**PHP version:**
7.1.14

**phpMyAdmin version:**
5.0.0-dev

**Comment by williamdes** [comment]
Issue confirmed in PMA 4.7.9
If I remember, there was a similar issue somewhere in PMA.It was related to using `==` instead of `===`
https://stackoverflow.com/a/2220571/5155484 or https://medium.com/@daveford/javascript-thinks-0-equals-empty-string-e40be4c7466a

### Demo server
![selection_023](https://user-images.githubusercontent.com/7784660/40334971-85b9ede8-5d61-11e8-8ffc-7fececd945ea.jpg)

**Comment by rijaspk** [comment]
when the form submits there is a check for ' ' (white spaces) but there isn't any check for 0

**Comment by rijaspk** [comment]
the issue occurred due to  using == instead of ===.
I will try to fix this.

**Comment by williamdes** [comment]
@rijaspk did you open a pull request ?

**Comment by ibennetch** [comment]
The error I get is:

```
{
    "pma_version": "4.8.2-dev",
    "browser_name": "FIREFOX",
    "browser_version": "60.0",
    "user_os": "Mac",
    "server_software": "Apache/2.4.25 (Debian)",
    "user_agent_string": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:60.0) Gecko/20100101 Firefox/60.0",
    "locale": "en",
    "configuration_storage": "enabled",
    "php_version": "7.2.5-1+0~20180505045740.21+stretch~1.gbpca2fa6",
    "exception_type": "js",
    "exception": {
        "mode": "stack",
        "name": "TypeError",
        "message": "dbStruct_url is undefined",
        "stack": [
            {
                "func": "?",
                "line": "125",
                "column": "17",
                "uri": "index.php?v=4.8.2-dev",
                "scriptname": "index.php"
            },
            {
                "func": "u",
                "line": "2",
                "column": "27452",
                "uri": "index.php?v=4.8.2-dev",
                "scriptname": "index.php"
            },
            {
                "func": "fireWith",
                "line": "2",
                "column": "28202",
                "uri": "index.php?v=4.8.2-dev",
                "scriptname": "index.php"
            },
            {
                "func": "k",
                "line": "2",
                "column": "77649",
                "uri": "index.php?v=4.8.2-dev",
                "scriptname": "index.php"
            },
            {
                "func": "n/<",
                "line": "2",
                "column": "79907",
                "uri": "index.php?v=4.8.2-dev",
                "scriptname": "index.php"
            }
        ],
        "uri": "index.php?"
    },
    "script_name": "index.php",
    "microhistory": nul
```

This affects both QA_4_8 and master.

**Comment by rijaspk** [comment]
@williamdes Apologies, Actually what happened is I referred to this issue in my commit by mistake. I didn't intend to create a pull request because the issue still there.

**Comment by williamdes** [comment]
@nulll-pointer +1

**Comment by ibennetch** [comment]
Fixed with the pull request #14398 from @nulll-pointer. The commit was cherry-picked to QA_4_8 as b0ec6b69e900814d4742c17f54eb7e2b57386ce6.

Thank you for reporting the issue! This will be fixed as part of the next release of phpMyAdmin.
